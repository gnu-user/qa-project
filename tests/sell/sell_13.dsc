TITLE
    Ticket Price Input Format (Edge case, value too large)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the sell 
    transaction. Enter a valid title, and enter invalid numerical input (a 
    double with two decimal places greater than 999.99) when prompted to enter 
    a ticket price. Verify that the input is not accepted and an error occurs.
DEPENDENCIES
    sell_9
