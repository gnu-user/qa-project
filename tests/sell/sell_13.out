Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the event title: 
Enter the sale price in dollars: 
Invalid sale price, amount cannot exceed 999.99.